packages/calcite-components/src/components/dialog/dialog.scss

```diff
@@ -0,0 +1,237 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-dialog-scrim-background: Specifies the background color of the component's scrim.
+ * @prop --calcite-dialog-width: Specifies the width of the component, using `px`, `em`, `rem`, `vw`, or `%`. Will never exceed the width of the viewport. Will not apply if `placement="cover"` if set.
+ * @prop --calcite-dialog-height: Specifies the height of the component, using `px`, `em`, `rem`, `vh`, or `%`. Will never exceed the height of the viewport. Will not apply if `placement="cover"` if set.
+ * @prop --calcite-dialog-panel-footer-padding: Specifies the padding of the component's footer.
+ * @prop --calcite-dialog-panel-header-border-block-end: Specifies the component header's block end border.
+ */
+
+:host {
+  --calcite-dialog-scrim-background: #{rgba($calcite-color-neutral-blk-240, $calcite-opacity-85)};
+  @apply flex inset-0 z-overlay pointer-events-none;
+  // the dialog should always use a dark scrim, regardless of light / dark mode - matches value in global.scss
+  --calcite-dialog-scrim-background-internal: #{rgba($calcite-color-neutral-blk-240, $calcite-opacity-85)};
+}
+
+:host([modal]) {
+  @apply absolute;
+}
+
+.container {
+  @apply text-color-2 pointer-events-auto
+    fixed
+    inset-0
+    flex
+    items-center
+    justify-center
+    overflow-y-hidden
+    opacity-0
+    z-overlay;
+  visibility: hidden;
+  transition:
+    visibility 0ms linear var(--calcite-internal-animation-timing-slow),
+    opacity var(--calcite-internal-animation-timing-slow) $easing-function;
+}
+
+:host([placement="top"]) .container {
+  @apply justify-center items-start;
+}
+
+:host([placement="top-start"]) .container {
+  @apply justify-start items-start;
+}
+
+:host([placement="top-end"]) .container {
+  @apply justify-end items-start;
+}
+
+:host([placement="bottom"]) .container {
+  @apply justify-center items-end;
+}
+
+:host([placement="bottom-start"]) .container {
+  @apply justify-start items-end;
+}
+
+:host([placement="bottom-end"]) .container {
+  @apply justify-end items-end;
+}
+
+:host(:not([modal])) .container {
+  @apply pointer-events-none;
+}
+
+:host([scale="s"]) {
+  --calcite-dialog-padding-internal: theme("spacing.3");
```

Comment:
Can you update these to use the naming schema?
`--calcite-internal-dialog-`

Reply:
Yep! thanks